src/elements/content-sidebar/Sidebar.js

```diff
@@ -102,13 +113,35 @@ class Sidebar extends React.Component<Props, State> {
             this.setForcedByLocation();
             this.setState({ isDirty: true });
         }
+
+        if (activeAnnotationId !== prevActiveAnnotationId) {
+            this.handleAnnotationSelect(activeAnnotationId);
+        }
     }
 
     getUrlPrefix = (pathname: string) => {
         const basePath = pathname.substring(1).split('/')[0];
         return basePath;
     };
 
+    /**
+     * Handle annotation selection
+     *
+     * @param {string} annotationId - The id of the annotation being selected
+     * @return {void}
+     */
+
+    handleAnnotationSelect = (annotationId: string | null): void => {
+        if (!annotationId) {
```

Comment:
I think if we return here, we leave the url as a deep-linked annotation which wouldn't be accurate. Should it instead set the url to be `/`?